src/languages/en.ts

```diff
@@ -3264,5 +3265,24 @@ export default {
             security: 'Expensify is PCI-DSS compliant, uses bank-level encryption, and utilizes redundant infrastructure to protect your data.',
             learnMoreAboutSecurity: 'Learn more about our security.',
         },
+        subscriptionSettings: {
+            title: 'Subscription settings',
+            autoRenew: 'Auto-renew',
+            yourAnnual: 'Your annual subscription will automatically renew on (date). You can switch to pay-per-use starting 30 days before renewal.',
+            autoIncrease: 'Auto-increase annual seats',
+            saveUpTo: 'Save up to $10/month per active member',
```

Comment:
This needs to be slightly variable: it should be `10` for a Collect plan and `18` for a Control plan. I'm also asking [in the doc](https://docs.google.com/document/d/1jXV_vIsOGZuL0dMiuXDWRETAFRX64ODVAhWPtiuFXT8/edit?disco=AAABPO4sn94) if it should always be in USD or not

Reply:
Just gonna tag you here directly too @MitchExpensify 😄

Reply:
I think this should be localised to the billing currency, personally. It doesn't make sense if you're billed in GBP.

Reply:
At the moment all prices on main Subscription page have fixed $ currency. Do you want me to fix them all as part of this PR or do we leave the currency as it is for now and change it later in a separate issue?

Reply:
I think it's fine to be a follow-up, but let's make sure we do it.

Reply:
Ok, so should we create new issue for it? If so, who's job is it? :)

Reply:
Polish GH for later follow-up: https://github.com/Expensify/App/issues/43277